src/sage/interfaces/magma_free.py

```diff
@@ -26,7 +26,7 @@ def magma_free_eval(code, strip=True, columns=0):
     Use the free online MAGMA calculator to evaluate the given
     input code and return the answer as a string.
 
-    LIMITATIONS: The code must evaluate in at most 20 seconds
+    LIMITATIONS: The code must evaluate in at most 120 seconds
```

Comment:
```suggestion
    .. WARNING::
    
        The code must evaluate in at most 120 seconds
```

Reply:
I think this is the proper way to warn the user of something like this: https://doc.sagemath.org/html/en/developer/coding_basics.html#the-docstring-of-a-function-content

(Also pushing another commit will re-run the CI and hopefully the error will go away)